frontend/src/scenes/insights/EmptyStates/EmptyStates.tsx

```diff
@@ -289,7 +290,7 @@ export function SavedInsightsEmptyState(): JSX.Element {
         <div className="saved-insight-empty-state">
             <div className="empty-state-inner">
                 <div className="illustration-main">
-                    <IconTrendUp />
+                    <BuilderHog3 />
                 </div>
```

Comment:
This is causing the hog to be full-size and I don't think he needs to be that giant 😄 Try something like:

```
<div className="illustration-main w-40">
    <BuilderHog3 className="w-full h-full" />
</div>
```

I'm technically using `w-50` but that class won't be available until my PR merges. You can use it here and just tolerate the hog being giant until my PR merges, or use `w-40` and he will be smaller than the other empty state ones.

Reply:
I think the big hog looks very friendly, but ok, squished him down:

<img width="820" alt="image" src="https://github.com/PostHog/posthog/assets/53387/dee82637-96ac-4edf-bd81-1b96b310a21e">